airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -118,6 +118,13 @@ class Config:
     ql: Optional[Literal[100, 200, 300, 400, 500, 600]] = None
 
 
+class Pypi(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    enabled: bool
+
```

Comment:
What do you think about adding the url to the pypi package?

Reply:
Seems in line with the "documentation url" and so on, added!

Reply:
Since our goal here is to publish, and tooling generally expects package name, could we call this the PyPi `package_name`?  I think that would be my vote. (E.g. `airbyte-source-apify` instead of `https://pypi.org/project/airbyte-source-apify/`)

PyPi URL just isn't very helpful because (to my knowlege at least), the full URL can't be passed to pip install or to the publish operation.

Fwiw, I do see "pip url" used frequently, but that is generally an alias for a PyPi package name or a Git ref, if the package isn't on PyPi.